airbyte-cdk/python/airbyte_cdk/sources/streams/concurrent/adapters.py

```diff
@@ -165,8 +208,7 @@ def get_json_schema(self) -> Mapping[str, Any]:
 
     @property
     def supports_incremental(self) -> bool:
-        # Only full refresh is supported
-        return False
+        return self._legacy_stream.supports_incremental
```

Comment:
The idea is that if a source is migrated to Concurrent CDK, due diligence is done to ensure that the incremental piece works fine. An example of this would be salesforce where [this](https://github.com/airbytehq/airbyte/blob/1ce84d6396e446e1ac2377362446e3fb94509461/airbyte-integrations/connectors/source-salesforce/source_salesforce/streams.py#L471) is not thread safe but we can't know from the code if it's properly supported or not